contents/blog/10x-engineers-do-user-interviews.md

```diff
@@ -0,0 +1,104 @@
+
+Then you need to can contact them and book a time. The simplest way is to email out a Calendly link in batches via bcc’ed. But quickly you should move to an in-app prompt. We’ve seen an increase in conversion from outreach to booking by ~5x (from 3% to 16%), a reduction in email spam, and more fine-grained control of the number of bookings a week. A reward like a $30 merchandise voucher helps. 
+
+Checkout PostHog’s [user interview app](https://posthog.com/tutorials/feedback-interviews-site-apps#setting-up-the-user-interview-app) for a quick way of showing prompts to segments of users or our [open-source set of components](https://github.com/PostHog/posthog-prompts/tree/luke/react-demo) for building your own.
+
+![User interview app](../images/blog/10x-engineers-do-user-interviews/user-interview-app.png)
+
+## Types of user interviews: problem exploration and solution exploration
+
+To get the most out of the interview, you should know the goal. The goal normally fits into two buckets:
+
+### 1. Problem exploration
+
+Find the biggest hair-on-fire problem and then deeply understand the parameters. Focus on the concrete times they’ve experienced the problem.
+
+Example questions:
+- How have they tried to solve it before? 
+- What do they think of the other solutions on the market? 
+- Would they pay for it? 
+- Can you talk me through the last time you experienced this problem? 
+- Is this actually a problem for you?
+
+**Recommendation:** Read "The Mom Test" by Rob Fitzpatrick. It's a fantastic primer on problem exploration.
+
+### 2. Solution validation 
+
+You’ve identified the hair-on-fire problem and understand it well. Now you’re validating whether your solution actually solves it. Focus on presenting the user with quick prototypes.
+
+Example activities:
+ - Have the user click through a Figma prototype and think out-loud. Ask them to complete the high-level goal and see where they get stuck. Ask what they think the text means.
+
+- Send a few mockups of the solutions over Slack to the user from the problem exploration interview and ask if it solves the problem.
+
+- Create a mock-up and see if the user will commit to paying some amount if you build it
+
+**Recommendation:** The book "Don’t make me think" is great for usability testing.
+
+## Beware solution ideas from users, reframe them as problems
+
+During interviews, users will often suggest an idea for the solution of what they want. It’s crucial to not accept this at face value and instead ask “Why?” until you can reframe it as a problem with the appropriate context. 
+
+Often the solution they propose doesn’t actually solve the problem they experienced, or there’s a much better solution that you can build instead. But most times it's not even a problem they are facing at all! Treat the solution idea as an ember and get curious about it.
```

Comment:
This is a great point. Are there any more common pitfalls worth covering? I think they'd add a lot of value.

Reply:
Will have a think and add some more